WARNING  pytest_result_log:plugin.py:122 Start: testcase/test07.py::test_sea_file_add_repo[8ac00e05a7652b31598a56d764bd5b5f7c45a6ef-a1-200-name]
WARNING  pytest_result_log:plugin.py:128 End: testcase/test07.py::test_sea_file_add_repo[8ac00e05a7652b31598a56d764bd5b5f7c45a6ef-a1-200-name]